tests/pyccel/test_pyccel.py

```diff
@@ -1084,6 +1084,12 @@ def test_function(language):
 
 #------------------------------------------------------------------------------
 @pytest.mark.xdist_incompatible
+@pytest.mark.parametrize( 'language', (
+        pytest.param("fortran", marks = pytest.mark.fortran),
+        pytest.param("c", marks = pytest.mark.c
+        )
+    )
+)
 @pytest.mark.xfail(os.environ.get('PYCCEL_DEFAULT_COMPILER', None) == 'intel', reason="1671")
```

Comment:
Has this PR broken the Python translation of inline methods? Can this be fixed?

Reply:
In Fortran and C, we don't print the inlined function, we can do something similar in Python , which we don't do for the moment

Reply:
Yes I think we don't do it at the moment so we match the interface that is exposed in Fortran and C. For C the function is inlined into the wrapper so it is still callable from Python. As discussed above you have changed this behaviour so the Python behaviour should be changed to match.

Reply:
[311ea3f](https://github.com/pyccel/pyccel/pull/1728/commits/311ea3f86c9a98c1a91b5d36991341bfc15cbd52)